cc/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Buf Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//cc:__subpackages__"])
+
+cc_test(
+    name = "grpc_client_test",
+    srcs = ["grpc_client_test.cc"],
+    env = {"PORT": "8081"},
```

Comment:
Just FYI, this seems to override env variables.

Reply:
Last I checked bazel ignored these when you use 'bazel run' , and uses them when you run 'bazel test' (which in turn ignores env)

Reply:
Here's what I saw:

```bash
$ bazel --version
6.3.0
$ PORT=9999 bazel run //cc:grpc_client_test
...
14: failed to connect to all addresses; last error: UNAVAILABLE: ipv4:127.0.0.1:8081: Socket closed
...
```

Just pointing out for anyone who might want to run it.

Reply:
Thanks for letting me know! (maybe the fixed it)